tests/containers/volumes.pm

```diff
@@ -0,0 +1,125 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: podman network
+# Summary: Test podman network
+# Maintainer: qac team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use version_utils qw(package_version_cmp);
+use containers::utils qw(get_podman_version);
+
+
+sub run {
+    my $runtime = get_required_var("CONTAINER_RUNTIME");
+
+    select_serial_terminal();
+
+    # From https://docs.docker.com/storage/bind-mounts/
+    # The --mount flag does not support z or Z options for modifying selinux labels.
+    my $Z = $runtime eq "podman" ? ",Z" : "";
+
+    my $test_dir = "test_dir";
+    my $test_file = "test_file";
+    my $test_image = "test_image";
+    my $test_container = "test_container";
+    my $test_volume = "test_volume";
+
+    assert_script_run("mkdir $test_dir");
+
+    # Create Dockerfile with VOLUME defined
+    assert_script_run("echo -e 'FROM busybox\nVOLUME /$test_dir' > $test_dir/Dockerfile");
+
+    # Build image
+    assert_script_run("$runtime build -t $test_image -f $test_dir/Dockerfile $test_dir/");
+
+    # Test --volumes-from
+    # Case 1: Check that the volume from container is visible in another container
+    assert_script_run("$runtime run -d --name $test_container $test_image");
+    assert_script_run("$runtime run --rm --volumes-from $test_container $test_image ls /$test_dir");
```

Comment:
```suggestion
    assert_script_run("$runtime run --rm --volumes-from $test_container $test_image stat /$test_dir/file");
```

`ls` for a directory is not sufficient to check if the volume is mounted. There are cases, where the mount point could be mounted as an empty volume.

Better to create a test file within `$test_dir` and check if that file is present.

Reply:
Fixed.